cost/google/idle_compute_instances/README.md

```diff
@@ -11,25 +11,41 @@ This Policy Template checks for idle instance in Google Compute Engine and then
 - If you get an **N/A** in a field you will need to install the [StackDriver Agent](https://cloud.google.com/monitoring/agent/install-agent) on the instance to get those metrics.
 - This policy only pulls running instances, as it is unable to get correct monitoring metrics from instances in other states
 
-### Input Parameters
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
-- *Google Cloud Project* - a Google Cloud Project name
 - *Average used memory percentage"* - Set to -1 to ignore memory utilization
 - *Average used CPU percentage* - Set to -1 to ignore CPU utilization
 - *Exclusion label Key:Value* - Cloud native label to ignore instances. Format: Key:Value
 
-### Cloud Management Required Permissions/Google Required Permissions
+## Actions
+
+The following policy actions are taken on any resources found to be out of compliance.
+
+- Send an email report
+- Delete all instances after approval
+
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `gce`
+
+Required permissions in the provider:
 
-- Cloud Management - The `credential_viewer`,`observer` roles
 - Google - The `Monitoring Viewer` Role, and the `compute.instances.delete`, `compute.instances.list`, `compute.instances.get` Permissions
+- Scope for the credential is <https://www.googleapis.com/auth/monitoring.write https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/monitoring.read https://www.googleapis.com/auth/monitoring https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/compute.readonly https://www.googleapis.com/auth/devstorage.full_control>
```

Comment:
No longer needed

Reply:
I have addressed this comment please review the changes.